docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ReplaceMissingValuesColumnOptions.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.ML.Data;
+using Microsoft.ML.SamplesUtils;
+using Microsoft.ML.Transforms;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class ReplaceMissingValuesColumnOptions
+    {
+        public static void Example()
+        {
+            // Creating the ML.Net IHostEnvironment object, needed for the pipeline.
+            var mlContext = new MLContext();
+
+            // Download the training and validation files.
+            string dataFile = DatasetUtils.DownloadMslrWeb10k();
+
+            // Create the loader to load the data.
+            var loader = mlContext.Data.CreateTextLoader(
+                columns: new[]
+                {
+                    new TextLoader.Column("Label", DataKind.Single, 0),
+                    new TextLoader.Column("GroupId", DataKind.String, 1),
+                    new TextLoader.Column("Features", DataKind.Single, new[] { new TextLoader.Range(2, 138) })
+                }
+            );
+
+            // Load the raw dataset.
+            var data = loader.Load(dataFile);
+            // Create the featurization pipeline. First, hash the GroupId column.
+            var pipeline = mlContext.Transforms.Conversion.Hash("GroupId")
+                // Replace missing values in Features column with the default replacement value for its type.
+                .Append(mlContext.Transforms.ReplaceMissingValues(new MissingValueReplacingEstimator.ColumnOptions("Features", "Features", MissingValueReplacingEstimator.ColumnOptions.ReplacementMode.Mean)));
+
+            // Fit the pipeline and transform the dataset.
+            var transformedData = pipeline.Fit(data).Transform(data);
```

Comment:
Please at least print out the columns affected before/after this transformation.

Reply:
+1

---
In reply to: [261021681](https://github.com/dotnet/machinelearning/pull/2773#discussion_r261021681) [](ancestors = 261021681)